interfaces/udev/backend.go

```diff
@@ -161,6 +161,18 @@ func (b *Backend) combineSnippets(snapInfo *snap.Info, snippets map[string][][]b
 		}
 	}
 
+	for _, slotInfo := range snapInfo.Slots {
+		securityTag := snap.NoneSecurityTag(snapInfo.Name(), slotInfo.Name)
```

Comment:
Let's please use this here instead:
```
nonePrefix := snap.NoneSecurityTag(snapInfo.Name(), "")
for securityTag, slotSnippets := range snippets {
        if !strings.HasPrefix(securityTag, nonePrefix) {
                continue
        }
        for _, snippet := range slotSnippets {
                snapSnippets[string(snippet)] = snippet
        }
}
```
This will pick both slots and plugs. We also need a test for this.